docs/fastdds/property_policies/non_consolidated_qos.rst

```diff
@@ -351,3 +351,72 @@ The behavior regarding this can be configured using the property ``fastdds.shm.e
            :language: xml
            :start-after: <!-->XML-SHM-ENFORCE-META-TRAFFIC
            :end-before: <!--><-->
+
+.. _property_max_message_size:
+
+Maximum Message Size
+^^^^^^^^^^^^^^^^^^^^
+
+It is possible to set the maximum number of bytes of an RTPS datagram sent by an RTPSParticipant.
+The value can be set in the RTPSParticipant properties or in the RTPSWriter properties.
```

Comment:
Even though the implementation is done at the RTPS level, we should refer to DDS entities here.

We should also add some introduction on the motivation. This property is useful for applications in which the values for the maximum size of a received datagram and a sent datagram should be different.

The configuration attribute in the transport is `maxMessageSize`, which is used both for reception and sending.
If there is an application where we want to receive big datagrams but a lower limit on the sent datagrams is required, then the new property can be used to set that lower limit.

We should explain here that this property will only lower the limit, i.e. it will only have an effect if set to a value lower than the minimum `maxMessageSize` on all transports.

Regarding the structure of the section:
1. Introduction on motivation and how it works (i.e. lower the limit on output datagrams imposed by `maxMessageSize`).
2. The table with name, type, and default value, along with the note of what happens when a wrong value is set.
3. A sub-section on how to set it up in a DomainParticipant.
4. A sub-section on how to set it up in a DataWriter.